### DISCUSSION TITLE
Workload= olap causes server disconnect

### DISCUSSION THREAD

**Discussion by end0cr1ne** [planetscale/discussion#541] [Bug Reports]
Running set workload=olap followed by a query now results in a server disconnection error within a few minutes.
I relied on the olap mode to run long queries and was surprised a few days ago to find that planetscale queries now failed if the world mode was set to olap.

I assume this is an unintended bug.

**Comment by orware** [comment]
Hello @end0cr1ne,

You mentioned this started happening a few days ago but I wanted to ask if you could narrow down the timeframe for when you started observing the behavior change?

Having a few more details or a code example of your current usage may be useful as well if it's able to be provided in a simplified example. You may also opt to submit more details via ticket instead if that may work better: https://support.planetscale.com/

**Reply by micahstone20** [reply]
This is happening to me as well.
